db/migrate/20230512094803_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :authorid
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Good job adding an __authorid__ as a column to represent a user, but I can see you are adding another reference at line 9 and this time it will create a __user_id__ column. I advise you to do research about creating custom foreign keys and learn how a user will be named author in other tables.

Hint:
 ```ruby
    t.references :author, foreign_key: { to_table: 'users' } # This will create author_id and a corresponding index 
```